tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -162,6 +163,10 @@ def bookmark(self, bookmark_title):
     def number_of_columns(self):
         return int(self.marionette.find_element(*self._landing_page_locator).get_attribute('cols'))
 
+    @property
+    def is_homescreen_visible(self):
```

Comment:
This looks fine to me, but I don't understand Johan's comment. It seems to be against the current revision of the code.